src/mongocxx/options/find_one_and_update.hpp

```diff
@@ -27,6 +26,17 @@ MONGOCXX_INLINE_NAMESPACE_BEGIN
 namespace options {
 
 ///
+/// Argument for find_one_and_update and find_one_and_replace to determine
+/// whether to return the document before or after modification.
+///
+/// @see https://docs.mongodb.org/manual/reference/method/db.collection.findAndModify/#return-data
+///
+enum class return_document {
```

Comment:
@samantharitter - is your suggestion to leave as?

I agreed with the original comment (thinking that it needed a new home). My opinion was that it belonged either in this file or in `find_one_and_replace.hpp`. As long as it's `mongocxx::options::return_document` and not `mongocxx::options::find_one_and_update::return_document`, this isn't something that a user would need to directly include anyway (you get all of this with collection.hpp), so it seemed okay to me.